contracts/RewardEscrowV2.sol

```diff
@@ -91,13 +92,17 @@ contract RewardEscrowV2 is
     /// @dev disable default constructor for disable implementation contract
     /// Actual contract construction will take place in the initialize function via proxy
     /// @custom:oz-upgrades-unsafe-allow constructor
-    constructor() {
+    constructor(address _kwenta) {
+        if (_kwenta == address(0)) revert ZeroAddress();
```

Comment:
I don't think this check is necessary, personally. If a bad address is given, you can always upgrade.

Reply:
Yeah I see where you are coming from. It is a little more gas to have the check, increases the deployment cost, but not by much. I am going to keep it for now. Probably won't follow this pattern in future though, it was really just to keep omniscia happy lol.

I think in the case of a revert it possibly might save some gas compared to fully deploying the contract 🤔.